app/src/main/java/org/covidwatch/android/data/CovidExposureInformation.kt

```diff
@@ -16,12 +17,16 @@ data class CovidExposureInformation(
     val id: Int = 0
 ) {
 
-    val howClose: Int
-        get() = when (attenuationValue) {
-            in 0..100 -> R.string.far_exposure_distance
-            in 101..200 -> R.string.close_exposure_distance
-            else -> R.string.near_exposure_distance
-        }
+    @Ignore
```

Comment:
Does this just mean deprecated? Not clear to me from documentation

Reply:
No, it means that we don't store these properties in the database. We ignore them :)